udata_front/theme/gouvfr/assets/js/pages/PublishingForm/PublishingForm.vue

```diff
@@ -154,6 +158,8 @@ const moveToStep = (step: number | null = null, saveToHistory = true) => {
 };
 
 onMounted(async () => {
+  const meValue = await fetchMe();
```

Comment:
I think this could be a simple promise with a then instead to avoid the delay before doing the granularities request